capone/tests/test_filter_by_related_objects.py

```diff
@@ -1,8 +1,7 @@
-from __future__ import unicode_literals
 from decimal import Decimal as D
 
 from django.test import TestCase
-from nose_parameterized import parameterized
+from parameterized import parameterized
```

Comment:
I think we can punt on this but should be using pytest's version in the future.

Reply:
I tried it, but it doesn't work on Django's TestCase methods.